server/src/main/java/org/apache/iotdb/db/service/thrift/impl/DataNodeInternalRPCServiceImpl.java

```diff
@@ -208,11 +246,13 @@ public TSendPlanNodeResp sendPlanNode(TSendPlanNodeReq req) {
   @Override
   public TFragmentInstanceStateResp fetchFragmentInstanceState(TFetchFragmentInstanceStateReq req) {
     FragmentInstanceId instanceId = FragmentInstanceId.fromThrift(req.fragmentInstanceId);
-    try (SetThreadName threadName = new SetThreadName(instanceId.getFullId())) {
```

Comment:
Why do we remove this line ?

Reply:
It seems that there is no log printed in this method.